packages/0018-package-format.md

```diff
@@ -0,0 +1,163 @@
+
+So if you start with a starter kit and then build upon document types to make a site, and then remove the package all content built on the starter kit document types would disappear.
+
+We want to make it safer and more transparent what happens when you uninstall a package, we have several ideas but it became a huge project on its own. So, for now, the main idea is to show the user a list of things that will be removed when they try to uninstall. See more ideas under the Out of Scope section below.
+
+### Other
+
+#### NuGet feed on Our
+
+We will create a NuGet feed on the Our website for packages uploaded to Our. This means any package in the new .nupkg package format that is uploaded to Our will be able to be consumed by a new custom NuGet feed. This custom NuGet feed can be consumed within Visual Studio or any development IDE by adding the Our NuGet feed URL to your NuGet.config file. The feed URL is yet to be determined.
+
+If developers wish to upload their packages to NuGet as well that is totally ok. Perhaps in the future there will be a way to automatically publish a package from Our to your NuGet account. There's no problem with having the same package/version in 2 different feeds.
+
+## Drawbacks
+
+Discuss any disadvantages or sideeffects of this RFC.
+
+## Alternatives
+
+- Use proprietary format for backoffice creation, installation and uninstallation, similar to what exists.
+
+## Out of Scope
+
+We have a few ideas that would be awesome to have but will be out of scope for this RFC:
+
+- Automatically check for upgrades and notify users with the right permissions
+- Check for references in other content / schema when trying to uninstall something and warn the user
+- Add option on uninstall to choose whether to uninstall the Umbraco content & schema
+- Use Umbraco Deploy as the serialization/deserialization engine
+
+## Unresolved Issues
+
+This RFC is mostly a feature plan for what we want to achieve with the first iteration of a new package format. Some of the features are on an early idea level and will need to be defined clearer before implementation can start. 
+
+- What packages do we list as installed in the backoffice? If a package is installed in the backoffice and it requires 10x NuGet packages that are not Umbraco NuGet packages but instead normal .NET libraries, do we list all of these? This would essentially mean we'd be listing everything that would exist in a `packages.config` file. If we don't list all of these then the backoffice user will have no way to uninstall these dependencies since we aren't going to uninstall a package's dependencies when it is uninstalled from the backoffice.
```

Comment:
Imho `packages.config` should not exist and should be use package reference instead. We should figure out the correct way of handling of project file with Umbraco, as project file contains a lot of additional pieces of information.

Reply:
See my comment... this was used as an example.